### PR TITLE
[SDFAB-1038] Add slice_id as configuration parameter and use it in the UP4 pipeline

### PR DESCRIPTION
**Pull request by osinstom**

### CODE REVIEW

File: conf/upf.json [hunk 1]
```diff
@@ -146,6 +146,9 @@
 
     "": "p4rtc interface settings",
     "p4rtciface": {
+        "": "Set the UP4 slice identifier that this PFCP Agent instance belongs to. Default: 0",
+        "slice_id": 0,
```
Comment (pudelkoM) [inline conf/upf.json:150]: It defaults to zero because `Unmarshal` init all fields to zero, right?

Comment (osinstom) [reply conf/upf.json:150]: Correct. I'll double-check with Daniele if 0 is a correct default value for `slice_id`.

Comment (pudelkoM) [reply conf/upf.json:150]: ok, do the usual manual init in `config.go` in that case (non-zero) then.

**Comment by osinstom** [comment]
@pudelkoM I confirmed with Daniele that it's ok to set 0 as default `slice_id`. Also, STC tests pass: https://jenkins.opencord.org/job/up4-pr-verify-pfcp/162/.

**Review by pudelkoM — APPROVED** [review]

**Pull request merged**